### PR TITLE
Added new Terraform Tool

### PR DESCRIPTION
**Pull request by kevsecops**
Added terraform-compliance as a tool for testing terraform code against security and policies

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -141,9 +141,11 @@ Infrastructure as Code allows applications to be deployed reliably to a consiste
 #### Terraform
 - [Checkov](https://github.com/bridgecrewio/checkov) - _Bridgecrew_ - Scan Terraform, AWS CloudFormation and Kubernetes templates for insecure configuration.
 - [Regula](https://github.com/fugue/regula) - _Fugue_ - Evaluate Terraform infrastructure-as-code for potential security misconfigurations and compliance violations prior to deployment.
+- [terraform compliance](https://terraform-compliance.com/) - _terraform-compliance_ - A lightweight, security and compliance focused test framework against terraform to enable negative testing capability for your infrastructure-as-code.
```
Comment (JakobTheDev) [inline readme.md:144]: Can you please change this to title case i.e. Terraform Compliance

File: readme.md [hunk 2]
```diff
@@ -141,9 +141,11 @@ Infrastructure as Code allows applications to be deployed reliably to a consiste
...
 - [Terrascan](https://github.com/cesar-rodriguez/terrascan) - _Cesar Rodriguez_ - Scan Terraform templates for best practice security configuration.
 - [Tfsec](https://github.com/liamg/tfsec) - _Liam Galvin_ - Scan Terraform templates for security misconfiguration and noncompliance with AWS, Azure and GCP security best practice.
 
+
```
Comment (JakobTheDev) [inline readme.md:148]: And remove this extra newline.

**Review by JakobTheDev — CHANGES_REQUESTED** [review]
Hi @kevsecops,

Thanks for the addition, just a couple of minor tweaks and we'll merge.

- Jakob

**Review by JakobTheDev — APPROVED** [review]
Thanks for the addition!

**Comment by kevsecops** [comment]
Thank you for the initial overview. It is really helpful!

**Pull request merged**